02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -111,9 +113,12 @@ If the concept of a "file path" is unfamiliar to you; we recommend taking a look
 
 # Using a different refine.bio dataset with this analysis?
 
-If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend placing the files in the `data/` directory you created and changing the filenames and paths in the notebook to match these files (we've put comments to signify where you would need to change the code).
-We suggest saving plots and results to `plots/` and `results/` directories, respectively, as these are automatically created by the notebook.
-From here you can customize this analysis example to fit your own scientific questions and preferences.
+If you'd like to adapt an example analysis to use a different dataset from [refine.bio](https://www.refine.bio/), we recommend replacing the `dge_url` with a different file path to a gene list that you are interested in. 
+
+If you have differential expression results or something with a gene-level ranking and a two-group comparison, we recommend considering [GSEA](https://alexslemonade.github.io/refinebio-examples/02-microarray/pathway-analysis_microarray_02_gsea.html) for your pathway analysis questions. 
```

Comment:
I think this sentence is likely to be missed here. I would move it to line 19, before you are getting into details about files and folders.

Reply:
Should I have it in both places?

Reply:
I left it in both places for now.

Reply:
I don't think it actually makes much sense here. The file you are describing has gene level ranking, so this caution seems more likely to confuse than illuminate when put here.

Reply:
I removed it in this instance but not the other one that I added in line 19.